.github/workflows/build.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     steps:
       # Pinned 1.0.0 version
-      - uses: haya14busa/action-workflow_run-status@967ed83efa565c257675ed70cfe5231f062ddd94
+      - uses: marocchino/action-workflow_run-status@54b6e87d6cb552fc5f36dbe9a722a6048725917a
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.workflow_run.head_branch }}
```

Comment:
Although unrelated to this PR, this should be `sha` instead of `branch`. Please fix.

Reply:
Updated the change